task/git-clone/0.2/README.md

```diff
@@ -0,0 +1,37 @@
+# git-clone task
+
+The git-clone Task will clone a repo from the provided url into the output Workspace. By default the repo will be cloned into the root of your Workspace.
+
+## Parameters
+|name|description|default value|required|
+|---|---|---|---|
+|url|Repository URL to clone from.||true|
+|revision|Revision to checkout. (branch, tag, sha, ref, etc...)|""|false|
+|refspec|Refspec to fetch before checking out revision.|""|false|
+|submodules|Initialize and fetch git submodules.|true|false|
+|depth|Perform a shallow clone, fetching only the most recent N commits.|1|false|
+|sslVerify|Set the `http.sslVerify` global git config. Setting this to `false` is not advised unless you are sure that you trust your git remote.|true|false|
+|subdirectory|Subdirectory inside the `output` Workspace to clone the repo into.|""|false|
+|sparseCheckoutDirectories|Define the directory patterns to match or exclude when performing a sparse checkout.|""|false|
+|deleteExisting|Clean out the contents of the destination directory if it already exists before cloning.|true|false|
+|httpProxy|HTTP proxy server for non-SSL requests.|""|false|
+|httpsProxy|HTTPS proxy server for SSL requests.|""|false|
+|noProxy|Opt out of proxying HTTP/HTTPS requests.|""|false|
+|verbose|Log the commands that are executed during `git-clone`'s operation.|true|false|
+|userHome|Absolute path to the user's home directory. Set this explicitly if you are running the image as a non-root user or have overridden the gitInitImage param with an image containing custom user configuration. |/tekton/home|false|
```

Comment:
We should drop the mention of the `gitInitImage` param in the description of `userHome`